src/pcapi/serialization/decorator.py

```diff
@@ -88,10 +91,13 @@ def decorate_validation(route: Callable[..., Any]) -> Callable[[Any], Any]:
         if 403 not in on_error_statuses:
             on_error_statuses.append(403)
 
-        spectree_response = SpectreeResponse(*[f"HTTP_{on_error_status}" for on_error_status in on_error_statuses])
+        response_codes = (
+            set([f"HTTP_{on_success_status}"])
+            | set(f"HTTP_{on_error_status}" for on_error_status in on_error_statuses)
+            | set(code_descriptions.keys())
+        )
 
-        if on_success_status == 204:
-            spectree_response.codes.append(f"HTTP_{on_success_status}")
+        spectree_response = SpectreeResponse(*(response_codes), code_descriptions=code_descriptions)
```

Comment:
(style / micro-détail)

```suggestion
        spectree_response = SpectreeResponse(*response_codes, code_descriptions=code_descriptions)
```

ça doit passer, non ?